.travis.yml

```diff
@@ -29,6 +29,7 @@ addons:
 
 install:
     - pip install -U pip tox codecov
+    - cd wapiti;python setup.py install;cd ..
```

Comment:
I think we shouldn't copy wapiti to webstruct repo. Also, it seems you've installed a very different wapiti: web application vulnerability scanner in Python, not a C++ CRF framework (https://wapiti.limsi.fr/).